client/src/pages/login/LoginPage.tsx

```diff
@@ -16,7 +15,6 @@ interface ILoginPageProps {
 export default function LoginPage({ goToSignUp }: ILoginPageProps) {
   const [user, setUser] = useState<LoginFormUser>({ email: "", password: "" });
   const { handleLogin } = useAuth();
-  const navigate = useNavigate();
```

Comment:
we can keep them on same page, might want to explore taking this off logout as well